### PR TITLE
Feat: new DS error loader

### PR DESCRIPTION
**Pull request by josenriagu**
## Description

- new DS ErrorLoader

## Checklist

- [x] I have read the **README** document
- [x] I have read the **CONTRIBUTING** document
- [ ] I have added tests to cover my changes
- [x] All new and existing tests passed
- [x] My commit message follows [Conventional Commit Guideline](https://conventionalcommits.org/)

### CODE REVIEW

File: ui/apps/app-center/src/components/pages/explore-page.tsx [hunk 1]
```diff
@@ -1,30 +1,26 @@
...
+const { Box, SubtitleTextIcon, DuplexButton, Icon, Spinner, NotificationPill, InfoCard } = DS;
+
+const { ErrorLoader } = DSNew;
 
 export interface IExplorePage extends RootComponentProps {
   installableApps: IntegrationReleaseInfoFragmentFragment[];
   installedAppsInfo?: IntegrationReleaseInfo[];
   isFetching?: boolean;
-  reqError?: Error;
+  reqError?: unknown;
```
Comment (quininez) [inline ui/apps/app-center/src/components/pages/explore-page.tsx]: why did you change the typing to unknown?

File: ui/apps/app-center/src/components/pages/info-page.tsx [hunk 2]
```diff
@@ -92,7 +95,7 @@ const InfoPage: React.FC<RootComponentProps> = props => {
           type="script-error"
           title={t('There was an error loading the integration info')}
           details={t('We cannot show this integration right now')}
-          devDetails={latestReleaseInfoReq.error}
+          devDetails={latestReleaseInfoReq.error as unknown as string}
```
Comment (quininez) [inline ui/apps/app-center/src/components/pages/info-page.tsx]: try to avoid typoe casting if possible. what is the original type of the error and why did it have to be changed?

**Review by quininez — CHANGES_REQUESTED** [review]

File: ui/apps/app-center/src/components/pages/info-page.tsx (hunk 2 above)
Comment (josenriagu) [reply ui/apps/app-center/src/components/pages/info-page.tsx]: Error is unknown, but the devDetails is of type string. I will find another way to fix this

**Review by quininez — APPROVED** [review]
